gradle/verification-metadata.xml

```diff
@@ -2033,6 +2033,76 @@
             <pgp value="34441e504a937f43eb0daef96a65176a0fb1cd0b"/>
          </artifact>
       </component>
+      <component group="org.codehaus.groovy" name="groovy-json" version="3.0.8">
```

Comment:
This whole addition should be replaced by us trusting the key for the `org.codehaus.groovy` group and cleaning up other entries that reference that key. Note that we have already such a section for a previous key value.